src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(e -> Arrays.stream(e.split(",")))
+                .map(Integer::parseInt)
+                .filter(e -> e % 2 == 0)
+                .min(Integer::compareTo)
```

Comment:
```suggestion
                .min()
```
It’s the default behavior